config-reloader/processors/relabel.go

```diff
@@ -28,7 +28,7 @@ func normalizeLabelName(ctx *ProcessorContext, label string) string {
 
 func (p *rewriteLabelsState) Process(input fluentd.Fragment) (fluentd.Fragment, error) {
 	normalizeAllLabels := func(d *fluentd.Directive, ctx *ProcessorContext) error {
-		if d.Name != "match" {
+		if d.Name != "match" && d.Name != "store" {
```

Comment:
This definitely fixes a deficiency and I think it has been reported before.
Can you please add a test that assert a label within a `store` gets processed? Like in this test which only tests the `match` https://github.com/vmware/kube-fluentd-operator/blob/master/config-reloader/processors/relabel_test.go#L75